src/pages/Post/Post.js

```diff
@@ -1,7 +1,203 @@
+import DaumPostcode from 'react-daum-postcode';
+import S from './Styled.Post';
 
 const Post = () => {
-  return <div>Post</div>;
+  const [isPopupOpen, setIsPopupOpen] = useState(false); // 팝업창 상태 관리
+  const [user, setUser] = useState({
+    title: '',
+    category: [],
+    hashTag: '',
+    zoneCode: '',
+    roadAddress: '',
+    detailAddress: '',
+    content: '',
+    x: '',
+    y: '',
+  });
+
+  let geocoder = new kakao.maps.services.Geocoder();
+
+  const callback = (result, status) => {
+    if (status === kakao.maps.services.Status.OK) {
+      setUser({ ...user, x: result[0].x, y: result[0].y });
+    }
+  };
+
+  useEffect(() => {
+    if (user.roadAddress.length > 0)
+      geocoder.addressSearch(`${user.roadAddress}`, callback);
+  }, [user.zoneCode]);
+
+  const onDrop = useCallback(acceptedFiles => {}, []);
+
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop }); // 업로드 drop
+
+  const onChange = e => {
+    const { name, value } = e.target;
+    setUser({ ...user, [name]: value });
+  };
+
+  const switchPostCode = () => {
+    if (!isPopupOpen) {
+      setUser({ ...user, zoneCode: '', roadAddress: '' });
+      setIsPopupOpen(prev => !prev);
+    } else {
+      setIsPopupOpen(prev => !prev);
+    }
+  };
+  const handlePostCode = data => {
+    setUser({
+      ...user,
+      zoneCode: data.zonecode,
+      roadAddress: data.roadAddress,
+    });
+    setIsPopupOpen(true);
+  };
+
+  const onCheckedElement = (checked, item) => {
+    if (checked) {
+      setUser({ ...user, category: user.category.concat(item) });
+    } else if (!checked) {
+      setUser({ ...user, category: user.category.filter(el => el !== item) });
+    }
+  };
+
+  return (
+    <div>
+      <S.Nav />
+      <S.Body>
+        <S.Inner>
+          <S.Head id="a">싱그런 여행 신청하기</S.Head>
+          <S.InputArea>
+            <S.InputTitle>
+              장소명<S.InputTitleSpan>(최대 30자)</S.InputTitleSpan>
+            </S.InputTitle>
+            <S.Input
+              type="text"
+              placeholder="예) 대구수목원"
+              name="title"
+              onChange={onChange}
+              maxLength="30"
+            />
+          </S.InputArea>
+          <S.CategoryArea>
+            <S.InputTitle>
+              카테고리 선택<S.InputTitleSpan>(다중 선택 가능)</S.InputTitleSpan>
+            </S.InputTitle>
+            <S.Category>
+              {CATEGORY_DATA.map(({ id, categoryName }) => {
+                return (
+                  <S.CategoryLabel key={id}>
+                    <S.CategoryCheckBox
+                      type="checkbox"
+                      value={categoryName}
+                      name="category"
+                      onChange={e => {
+                        onCheckedElement(e.target.checked, e.target.value);
+                      }}
+                      checked={
+                        user.category.includes(categoryName) ? true : false
+                      }
```

Comment:
includes 메서드 자체가 boolean을 반환하기 때문에 삼항연산자에서 사용하신 조건 자체를 사용하시면 됩니다.
```suggestion
                      checked={
                        user.category.includes(categoryName)
                      }
```